apps/communications/contacts/locales/contacts.en-US.properties

```diff
@@ -15,7 +15,7 @@ selectContact      = Select contact
 selectAll          = Select all
 deselectAll        = Deselect all
 editContact        = Edit contact
-deleteConfirmMsg   = Delete contact?
+deleteConfirmMsg   = Are you sure you want to delete contact?
```

Comment:
As mentioned by David in comments for initial PR of Settings to Mozilla 2.2r branch (see https://github.com/mozilla-b2g/gaia/pull/32750) you shall not change string without assigning new l10n id. If you need different text you **shall also define new l10nid** for it, otherwise localization team will not detect this change and text will remain untranslated.
One more point, as soon as you updating this localization file, probably it will be better to fix remaining David's comments related to this file (e.g. lines 115-117), since without having them fixed we won't be allowed to land this code.